tetrateci/test_1.9.sh

```diff
@@ -2,11 +2,13 @@
 set -e
 
 git apply tetrateci/patches/common/disable-dashboard.1.9.patch
-git apply tetrateci/patches/common/disable-multicluster.1.9.patch
 git apply tetrateci/patches/common/disable-ratelimiting.1.9.patch
-git apply tetrateci/patches/common/disable-vmospost.1.9.patch
 git apply tetrateci/patches/common/disable-stackdriver.1.9.patch
 
+if $(grep -q "1.17" <<< ${VERSION} ); then
+  git apply tetrateci/patches/common/disable-endpointslice.1.8.patch
```

Comment:
Is this patch same across 1.8 and 1.9? If so, we could rename the file to something generic and add a comment on why we do this. WDYT?

Reply:
ahh I forgot to port back this commit from my testing branch to change the name, you can actually see that I have a copy of the same patch included in the pr